logback-core/src/test/java/ch/qos/logback/core/rolling/SizeAndTimeBasedFNATP_Test.java

```diff
@@ -193,4 +193,35 @@ public void withZipCompression_FileSet_NoRestart_7() throws Exception {
     List<String> zipFiles = filterElementsInListBySuffix(".zip");
     checkZipEntryMatchesZipFilename(zipFiles);
   }
+
+  @Test
+  public void rotateIfFileRemoved_Fileset_NoRestart_8() throws Exception {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/tony19/logback/file/2371511790/issues/source?bid=3222618&fileBranchId=3222674#l198)